package.json

```diff
@@ -33,6 +33,7 @@
     "babel-eslint": "^6.1.2",
     "babel-jest": "^15.0.0",
     "babel-loader": "^6.2.5",
+    "babel-plugin-rewire": "^1.0.0",
```

Comment:
We should be able to do this with `jest` as well via [manual mocks](https://facebook.github.io/jest/docs/manual-mocks.html) for this test case.